packages/tasks_panel/src/commands/reveal-task.ts

```diff
@@ -0,0 +1,42 @@
+import { Range, Selection, Uri, window } from "vscode";
+import { getLogger } from "../logger/logger-wrapper";
+import { TaskTreeItem } from "../view/task-tree-item";
+import { messages } from "../i18n/messages";
+import { disposeTaskEditorPanel, getTaskEditor } from "../panels/panels-handler";
+import { size } from "lodash";
+
+export async function revealTask(treeItem: TaskTreeItem): Promise<void> {
+  try {
+    if (treeItem.command?.arguments === undefined) {
+      return;
+    }
+    const task = treeItem.command.arguments[0];
```

Comment:
can arguments be an empty array?

Reply:
not (else it is wrong tree item). but anyhow I prefer to get an exception in that case and display it